drivers/adxl345/adxl345.c

```diff
@@ -34,7 +34,8 @@
 #define BUS                 (dev->params.i2c)
 #define ADDR                (dev->params.addr)
 
-int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
+int adxl345_init(adxl345_t *dev, const adxl345_params_t* params, uint8_t range,
+                 uint8_t rate, uint8_t full_res, float scale_factor)
```

Comment:
As stated in https://github.com/RIOT-OS/RIOT/wiki/Guide:-Writing-a-device-driver-in-RIOT, the init function should have the `dev` and `params` struct as only parameters...